sdk/vaa/structs.go

```diff
@@ -265,6 +265,38 @@ func ChainIDFromString(s string) (ChainID, error) {
 	}
 }
 
+func GetAllNetworkIDs() []ChainID {
+	return []ChainID{
+		ChainIDSolana,
+		ChainIDEthereum,
+		ChainIDTerra,
+		ChainIDBSC,
+		ChainIDPolygon,
+		ChainIDAvalanche,
+		ChainIDAlgorand,
+		ChainIDAptos,
+		ChainIDSui,
+		ChainIDOasis,
+		ChainIDAurora,
+		ChainIDFantom,
+		ChainIDKarura,
+		ChainIDAcala,
+		ChainIDKlaytn,
+		ChainIDCelo,
+		ChainIDNear,
+		ChainIDTerra2,
+		ChainIDPythNet,
+		ChainIDMoonbeam,
+		ChainIDArbitrum,
+		ChainIDOptimism,
+		ChainIDXpla,
+		ChainIDBtc,
+		ChainIDInjective,
+		ChainIDNeon,
+		ChainIDWormchain,
```

Comment:
Can we keep this list in the same order as the consts below?  That will make it easier to see if any are missing.

Reply:
done